config/apprunner/config.go

```diff
@@ -0,0 +1,28 @@
+/*
+Copyright 2022 Upbound Inc.
+*/
+
+package apprunner
+
+import (
+	"github.com/upbound/upjet/pkg/config"
+)
+
+// Configure adds configurations for apprunner group.
+func Configure(p *config.Provider) {
+	p.AddResourceConfigurator("aws_apprunner_vpc_connector", func(r *config.Resource) {
+		r.References = map[string]config.Reference{
```

Comment:
Please configure new references instead of overriding (possible) previous reference configuration.

Reply:
I have added new reference configurations, thanks.